### ISSUE TITLE
add the storybook interactions add-on and accessibility add-on

### ISSUE THREAD

**Issue by Kira-Pilot** [coder/coder#11422]
We are interested in adding the [storybook interactions add-on](https://storybook.js.org/docs/writing-tests/interaction-testing) to our FE as a possible replacement for jest testing. We should try this out and write a couple of tests as a POC.

Additionally we want to try out the [storybook accessibility add-on,](https://github.com/storybookjs/storybook) so we should add this as well, along with a couple of tests. In the past it was uncertain if this was playing nicely with Chromatic so this should be part of the discovery work.

**Comment by aslilac** [comment]
We're going to keep the accessibility addon on our radar, but the Chromatic integration is lacking, and it doesn't seem like much of a win right now. The interactions addon has been added and seems good!
